random_blink/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM resin/rpi-raspbian
+
+# Install Go 1.7.5 and enough of a build-chain to build wiringpi (in C)
+RUN apt-get update && \
+    apt-get install -qy build-essential wiringpi git curl ca-certificates && \
+    curl -sSLO https://storage.googleapis.com/golang/go1.7.5.linux-armv6l.tar.gz && \
+	mkdir -p /usr/local/go && \
+	tar -xvf go1.7.5.linux-armv6l.tar.gz -C /usr/local/go/ --strip-components=1
+
+ENV PATH=$PATH:/usr/local/go/bin/
+ENV GOPATH=/go/
+
+RUN mkdir -p /go/src/github.com/alexellis/blinkt_go_examples/random_blink
+WORKDIR /go/src/github.com/alexellis/blinkt_go_examples/random_blink
+
+COPY app.go	.
+RUN go get -d -v
+
+RUN go build
+
+CMD ["./random_blink"]
```

Comment:
There are pros/cons, but I think it will be too much maintenance for us. I was also thinking of making the app name more generic in the container so that it's even easier to maintain.

Source folder `/src/github.com/alexellis/blinkt_go_examples/app` for instance and compile to a name of `app` so all the `cmd` instructions are the same. The `copy` could be `COPY . .`.

I will merge this part as-is, but we should refactor the duplication out soon.